app/templates/index/mygoals.hbs

```diff
@@ -1,5 +1,5 @@
-{{mygoals/selection-goals model=model type="working"}}
+{{component 'bepstore-goals@goal/goal-mine' model=model type="working"}}
```

Comment:
I think the "goal-mine" functionality is more appropriate in bepstore-ui than in ui-goals
Only bepstore-ui uses this functionality and it is not relevant for fbf to have it in the goal